front/components/Avatar/avatar.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+
+import Avatar from './index';
+
+const mockupUser = {
+  id: '414',
+  username: 'Toto',
+  email: 'toto@gmail.com',
+  description: "I'm an example",
+  packages: [],
+};
+
+test('Avatar T first letter', () => {
+  render(<Avatar user={mockupUser} />);
+
+  const avatar = screen.getByTestId('user-avatar');
+  expect(avatar).toHaveTextContent('T');
+});
+
+const mockupUser2 = {
+  id: '414',
+  username: 'Bobo',
+  email: 'bobo@gmail.com',
+  description: "I'm an example",
+  packages: [],
+};
+
+test('Avatar B first letter', () => {
+  render(<Avatar user={mockupUser2} />);
+
+  const avatar = screen.getByTestId('user-avatar');
+  expect(avatar).toHaveTextContent('B');
+});
```

Comment:
Looks like both tests are testing the same thing. I think we could keep only one